### PR TITLE
Questionnaire

### PR DESCRIPTION
**Pull request by adwilson07**
Small refactor to the structure and new test for questionnaires.

### CODE REVIEW

File: src/main/groovy/PostgressAccess/PGAccess.groovy [hunk 1]
```diff
@@ -0,0 +1,158 @@
+package PostgressAccess;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.sql.*
+
```
Comment (bryan-laipple-thinktank) [inline src/main/groovy/PostgressAccess/PGAccess.groovy]: Using objects from java.sql can be cumbersome.  We might want to investigate using something to allow us to be less verbose.  I found sql2o while working visual-service-tests and liked it.  I had never used it before but took about an hour to learn.

File: src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	}
+	
+	
+	
+	//
+	// Selectors here!!!!
+	// constantly subject to verification and change
+	//
+	@FindBy(xpath = "//*[contains(@class,'CreateActivity')]//button") private WebElement addActivity
+	@FindBy(xpath = "//create-activity-modal/div[2]/button[2]") private WebElement createQuestionnaireButton
```
Comment (kycin) [inline src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy]: ug, usually don't we want to stay away from patterns like this (...[n]...[m]...)? Or is there simply no choice here?

File: qa.properties [hunk 3]
```diff
@@ -16,4 +16,14 @@ webdriver.chrome.win=webdrivers/win/chromedriver
...
+
+
+#Postgres DB
+#QA
+postgresURL=jdbc:postgresql://qa.cnmxoe3fmzkf.us-west-2.rds.amazonaws.com/thinktank?currentSchema=project_service
+
+
+postgresUser=postgres
+postgresPassword=postgres
+
```
Comment (kycin) [inline qa.properties]: please add same parameters to dev.properties, just to maintain the properties files on same footing

File: build.gradle [hunk 4]
```diff
@@ -18,6 +18,7 @@ dependencies {
     compile group: 'info.cukes', name: 'cucumber-core', version: '1.2.5'
     compile group: 'info.cukes', name: 'cucumber-groovy', version: '1.2.5'
     compile group: 'info.cukes', name: 'cucumber-html', version: '0.2.3'
+	compile group: 'info.cukes', name: 'cucumber-junit', version: '1.2.5'
```
Comment (kycin) [inline build.gradle]: not sure why is this necessary, cucumber-junit

File: src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy (hunk 2 above)
Comment (kycin) [inline src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy]: ug, usually we want to avoid this pattern for selectors (...[n]...); perhaps there's no choice in this context?

File: src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+	
+	public Tt7ClientActivityPage clickCreateQuestionnaireActivityButton() {
+		 this.clickElement(createQuestionnaireActivityButton)
+		 return this
+	}
+	
+	public Tt7ClientActivityPage clickQuestionnairePublishButton() {
+		
+		String questionId = getQuestionnaireId()
+		questionId = "//*[@id='$questionId']//button[1]"
```
Comment (kycin) [inline src/main/groovy/net/thinktank/test/selenium/page/Tt7ClientActivityPage.groovy]: let's avoid putting selector string inline in method code, keep all in one place at top of class, store in static final strings

File: src/test/groovy/features/Questionnaire_Activity.feature [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Feature: QuestionnaireFeature
+  Login to home page of TT7 client
+
+  @Questionnaire
+  Scenario: I can create and publish a questionnaire
+    Given I login with user name xxxx and password yyyyy
```
Comment (kycin) [inline src/test/groovy/features/Questionnaire_Activity.feature]: oops, this was only a placeholder, let's get something more professional looking than the xxxx, yyyy
